src/ts/state/reducer.ts

```diff
@@ -563,7 +563,7 @@ const amendBlockQueriesInFlight = (
   if (!currentBlockQueriesInFlight) {
     return state.requestsInFlight;
   }
-  const newBlockQueriesInFlight: IBlocksInFlightState = {
+  const newBlockQueriesInFlight: IRequestInFlight = {
```

Comment:
Just want to check - are we still satisfied with the variable name `newBlockQueriesInFlight` given we have renamed the typing?

Reply:
Otherwise, LGTM!

Reply:
Great catch – no we are not! I've amended that, and updated another small shibboleth left over from an early name change (blockQueriesInFlight should just be blocksInFlight).